src/components/Scores/Score.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import InputCounter from "./InputCounter";
+import styles from './Score.module.css';
+
+interface IProps {
```

Comment:
Props also should be exported because they are the part of a component signature.

Reply:
What's the profit of extracting props interface of component? I don't use them anywhere outside it

Reply:
Because it is the part of component signature, so it is public.